### ISSUE TITLE
Error when running GRIDSS

### ISSUE THREAD

**Issue by GiterKR** [PapenfussLab/gridss#474]
Hi,

Please help me in resolving the error I get when running GRIDSS.

Here is the command I used:
```
../../gridss.sh -r ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa \
				-o VCF/BH004_grids.vcf.gz \
				-a BAM/BH004_grids.bam \
				--labels BH004 \
				../BAM/BH004_ReSorted.bam 
```

Here is the output displayed:
```
Fri Mar 19 03:15:27 EDT 2021: Full log file is: ./gridss.full.20210319_031527.node155.hpc.local.301999.log
which: no time in (/usr/bin)
Fri Mar 19 03:15:27 EDT 2021: Not found /usr/bin/time
Fri Mar 19 03:15:27 EDT 2021: Using GRIDSS jar /home/rajk/gridss-2.11.0-gridss-jar-with-dependencies.jar
Fri Mar 19 03:15:27 EDT 2021: Using reference genome "../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa"
Fri Mar 19 03:15:27 EDT 2021: Using output VCF VCF/BH004_grids.vcf.gz
Fri Mar 19 03:15:27 EDT 2021: Using assembly bam  BAM/BH004_grids.bam
Fri Mar 19 03:15:27 EDT 2021: Using 8 worker threads.
Fri Mar 19 03:15:27 EDT 2021: Using no blacklist bed. The encode DAC blacklist is recommended for hg19.
Fri Mar 19 03:15:27 EDT 2021: Using JVM maximum heap size of 30g for assembly and variant calling.
Fri Mar 19 03:15:27 EDT 2021: Using input file ../BAM/BH004_ReSorted.bam
Fri Mar 19 03:15:27 EDT 2021: label is BH004
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/R/4.0.2/bin/Rscript
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/samtools/1.11/bin/samtools
Fri Mar 19 03:15:27 EDT 2021: Found /etc/alternatives/java_sdk_1.8.0/bin/java
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/bwa/0.7.10/bin/bwa
Fri Mar 19 03:15:27 EDT 2021: samtools version: 1.11+htslib-1.11
Fri Mar 19 03:15:27 EDT 2021: R version: R scripting front-end version 4.0.2 (2020-06-22)
Fri Mar 19 03:15:27 EDT 2021: bwa Version: 0.7.10-r789
which: no time in (/usr/bin)
Fri Mar 19 03:15:27 EDT 2021: bash version: GNU bash, version 4.2.46(2)-release (x86_64-redhat-linux-gnu)
Fri Mar 19 03:15:28 EDT 2021: java version: openjdk version "1.8.0_242" OpenJDK Runtime Environment (build 1.8.0_242-b08)       OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode)
Fri Mar 19 03:15:28 EDT 2021: Max file handles: 4096
Fri Mar 19 03:15:28 EDT 2021: Running GRIDSS steps: setupreference, preprocess, assemble, call,
Fri Mar 19 03:15:28 EDT 2021: Start pre-processing      ../BAM/BH004_ReSorted.bam
Fri Mar 19 03:15:28 EDT 2021: Running   CollectGridssMetrics    ../BAM/BH004_ReSorted.bam       first 10000000 records
"$timecmd java -Xmx$otherjvmheap $jvm_args -cp $gridss_jar gridss.analysis.CollectGridssMetrics REFERENCE_SEQUENCE=$reference TMP_DIR=$dir ASSUME_SORTED=true I=$f O=$tmp_prefix THRESHOLD_COVERAGE=$maxcoverage FILE_EXTENSION=null GRIDSS_PROGRAM=null GRIDSS_PROGRAM=CollectIdsvMetrics PROGRAM=null PROGRAM=CollectInsertSizeMetrics STOP_AFTER=$metricsrecords $picardoptions" command completed with exit code 1.
*****
The underlying error message can be found in ./gridss.full.20210319_031527.node155.hpc.local.301999.log.
*****
```

Here is the text in the log file:
```
Fri Mar 19 03:15:27 EDT 2021: Full log file is: ./gridss.full.20210319_031527.node155.hpc.local.301999.log
Fri Mar 19 03:15:27 EDT 2021: Not found /usr/bin/time
Fri Mar 19 03:15:27 EDT 2021: Using GRIDSS jar /home/rajk/gridss-2.11.0-gridss-jar-with-dependencies.jar
Fri Mar 19 03:15:27 EDT 2021: Using reference genome "../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa"
Fri Mar 19 03:15:27 EDT 2021: Using output VCF VCF/BH004_grids.vcf.gz
Fri Mar 19 03:15:27 EDT 2021: Using assembly bam  BAM/BH004_grids.bam
Fri Mar 19 03:15:27 EDT 2021: Using 8 worker threads.
Fri Mar 19 03:15:27 EDT 2021: Using no blacklist bed. The encode DAC blacklist is recommended for hg19.
Fri Mar 19 03:15:27 EDT 2021: Using JVM maximum heap size of 30g for assembly and variant calling.
Fri Mar 19 03:15:27 EDT 2021: Using input file ../BAM/BH004_ReSorted.bam
Fri Mar 19 03:15:27 EDT 2021: label is BH004
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/R/4.0.2/bin/Rscript
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/samtools/1.11/bin/samtools
Fri Mar 19 03:15:27 EDT 2021: Found /etc/alternatives/java_sdk_1.8.0/bin/java
Fri Mar 19 03:15:27 EDT 2021: Found /opt/software/bwa/0.7.10/bin/bwa
Fri Mar 19 03:15:27 EDT 2021: samtools version: 1.11+htslib-1.11
Fri Mar 19 03:15:27 EDT 2021: R version: R scripting front-end version 4.0.2 (2020-06-22)
Fri Mar 19 03:15:27 EDT 2021: bwa Version: 0.7.10-r789
Fri Mar 19 03:15:27 EDT 2021: bash version: GNU bash, version 4.2.46(2)-release (x86_64-redhat-linux-gnu)
Fri Mar 19 03:15:28 EDT 2021: java version: openjdk version "1.8.0_242" OpenJDK Runtime Environment (build 1.8.0_242-b08)       OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode) 
Fri Mar 19 03:15:28 EDT 2021: Max file handles: 4096
Fri Mar 19 03:15:28 EDT 2021: Running GRIDSS steps: setupreference, preprocess, assemble, call,
Fri Mar 19 03:15:28 EDT 2021: Start pre-processing      ../BAM/BH004_ReSorted.bam
Fri Mar 19 03:15:28 EDT 2021: Running   CollectGridssMetrics    ../BAM/BH004_ReSorted.bam       first 10000000 records
INFO    2021-03-19 03:15:28     Defaults        Found file for property samjdk.reference_fasta: /home/rajk/SV_MAC_BH/GRIDSS/../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa 
INFO    2021-03-19 03:15:28     CollectGridssMetrics    

********** NOTE: Picard's command line syntax is changing.
**********
********** For more information, please see:
********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)
**********
********** The command line looks like this in the new syntax:
**********
**********    CollectGridssMetrics -REFERENCE_SEQUENCE ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa -TMP_DIR ./BH004_ReSorted.bam.gridss.working -ASSUME_SORTED true -I ../BAM/BH004_ReSorted.bam -O ./BH004_ReSorted.bam.gridss.working/tmp.BH004_ReSorted.bam -THRESHOLD_COVERAGE 50000 -FILE_EXTENSION null -GRIDSS_PROGRAM null -GRIDSS_PROGRAM CollectIdsvMetrics -PROGRAM null -PROGRAM CollectInsertSizeMetrics -STOP_AFTER 10000000
**********


03:15:28.683 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rajk/gridss-2.11.0-gridss-jar-with-dependencies.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Mar 19 03:15:28 EDT 2021] CollectGridssMetrics GRIDSS_PROGRAM=[CollectIdsvMetrics] THRESHOLD_COVERAGE=50000 INPUT=../BAM/BH004_ReSorted.bam ASSUME_SORTED=true STOP_AFTER=10000000 OUTPUT=./BH004_ReSorted.bam.gridss.working/tmp.BH004_ReSorted.bam FILE_EXTENSION=null PROGRAM=[CollectInsertSizeMetrics] TMP_DIR=[./BH004_ReSorted.bam.gridss.working] REFERENCE_SEQUENCE=../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa    METRIC_ACCUMULATION_LEVEL=[ALL_READS] INCLUDE_UNPAIRED=false VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
[Fri Mar 19 03:15:28 EDT 2021] Executing as rajk@node155 on Linux 3.10.0-1127.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_242-b08; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.11.0-gridss
ERROR   2021-03-19 03:15:29     ReferenceCommandLineProgram     Reference genome used by ../BAM/BH004_ReSorted.bam does not match reference genome ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa. The reference supplied must match the reference used for every input.
[Fri Mar 19 03:15:29 EDT 2021] gridss.analysis.CollectGridssMetrics done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=2075918336
Exception in thread "main" htsjdk.samtools.util.SequenceUtil$SequenceListsDifferException: In files /home/rajk/SV_MAC_BH/GRIDSS/../BAM/BH004_ReSorted.bam and /home/rajk/SV_MAC_BH/GRIDSS/../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa
        at htsjdk.samtools.util.SequenceUtil.assertSequenceDictionariesEqual(SequenceUtil.java:345)
        at gridss.cmdline.ReferenceCommandLineProgram.ensureDictionaryMatches(ReferenceCommandLineProgram.java:117)
        at gridss.analysis.CollectGridssMetrics.doWork(CollectGridssMetrics.java:75)
        at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:305)
        at picard.cmdline.CommandLineProgram.instanceMainWithExit(CommandLineProgram.java:196)
        at gridss.analysis.CollectGridssMetrics.main(CollectGridssMetrics.java:57)
Caused by: htsjdk.samtools.util.SequenceUtil$SequenceListsDifferException: Sequences at index 1 don't match: 1/69331447/10 1/85426708/2/M5=526c549b204117f61cd292042a7127d2/UR=file:/home/rajk/SV_MAC_BH/GRIDSS/../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa
        at htsjdk.samtools.util.SequenceUtil.assertSequenceListsEqual(SequenceUtil.java:272)
        at htsjdk.samtools.util.SequenceUtil.assertSequenceDictionariesEqual(SequenceUtil.java:334)
        at htsjdk.samtools.util.SequenceUtil.assertSequenceDictionariesEqual(SequenceUtil.java:320)
        at htsjdk.samtools.util.SequenceUtil.assertSequenceDictionariesEqual(SequenceUtil.java:343)
        ... 5 more
```

Appreciate all your help.

**Comment by alexiswl** [comment]
I think @d-cameron can comment on if gridss is strictly tinkered for human genomes only. 

The error in this case it looks like your bam file was aligned with a reference that is not the input to your gridss command.  

If you run `cat ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa | grep '>'` you'll see the list of contigs of the reference file. You can also run `samtools view -H ../BAM/BH004_ReSorted.bam | grep '@SQ'` to see the list of contigs of the reference file that was used to generated the aligned bam file.  Hope that helps!

**Comment by GiterKR** [comment]
Hi @alexiswl 

Thank you for looking into this. I ran those commands to verify the contigs in the reference and BAM file, they look like they match, see below.

Here is the first 45 lines of the output for `cat ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa | grep '>'`
```
>1 dna:chromosome chromosome:CanFam3.1:1:1:122678785:1 REF
>2 dna:chromosome chromosome:CanFam3.1:2:1:85426708:1 REF
>3 dna:chromosome chromosome:CanFam3.1:3:1:91889043:1 REF
>4 dna:chromosome chromosome:CanFam3.1:4:1:88276631:1 REF
>5 dna:chromosome chromosome:CanFam3.1:5:1:88915250:1 REF
>6 dna:chromosome chromosome:CanFam3.1:6:1:77573801:1 REF
>7 dna:chromosome chromosome:CanFam3.1:7:1:80974532:1 REF
>8 dna:chromosome chromosome:CanFam3.1:8:1:74330416:1 REF
>9 dna:chromosome chromosome:CanFam3.1:9:1:61074082:1 REF
>10 dna:chromosome chromosome:CanFam3.1:10:1:69331447:1 REF
>11 dna:chromosome chromosome:CanFam3.1:11:1:74389097:1 REF
>12 dna:chromosome chromosome:CanFam3.1:12:1:72498081:1 REF
>13 dna:chromosome chromosome:CanFam3.1:13:1:63241923:1 REF
>14 dna:chromosome chromosome:CanFam3.1:14:1:60966679:1 REF
>15 dna:chromosome chromosome:CanFam3.1:15:1:64190966:1 REF
>16 dna:chromosome chromosome:CanFam3.1:16:1:59632846:1 REF
>17 dna:chromosome chromosome:CanFam3.1:17:1:64289059:1 REF
>18 dna:chromosome chromosome:CanFam3.1:18:1:55844845:1 REF
>19 dna:chromosome chromosome:CanFam3.1:19:1:53741614:1 REF
>20 dna:chromosome chromosome:CanFam3.1:20:1:58134056:1 REF
>21 dna:chromosome chromosome:CanFam3.1:21:1:50858623:1 REF
>22 dna:chromosome chromosome:CanFam3.1:22:1:61439934:1 REF
>23 dna:chromosome chromosome:CanFam3.1:23:1:52294480:1 REF
>24 dna:chromosome chromosome:CanFam3.1:24:1:47698779:1 REF
>25 dna:chromosome chromosome:CanFam3.1:25:1:51628933:1 REF
>26 dna:chromosome chromosome:CanFam3.1:26:1:38964690:1 REF
>27 dna:chromosome chromosome:CanFam3.1:27:1:45876710:1 REF
>28 dna:chromosome chromosome:CanFam3.1:28:1:41182112:1 REF
>29 dna:chromosome chromosome:CanFam3.1:29:1:41845238:1 REF
>30 dna:chromosome chromosome:CanFam3.1:30:1:40214260:1 REF
>31 dna:chromosome chromosome:CanFam3.1:31:1:39895921:1 REF
>32 dna:chromosome chromosome:CanFam3.1:32:1:38810281:1 REF
>33 dna:chromosome chromosome:CanFam3.1:33:1:31377067:1 REF
>34 dna:chromosome chromosome:CanFam3.1:34:1:42124431:1 REF
>35 dna:chromosome chromosome:CanFam3.1:35:1:26524999:1 REF
>36 dna:chromosome chromosome:CanFam3.1:36:1:30810995:1 REF
>37 dna:chromosome chromosome:CanFam3.1:37:1:30902991:1 REF
>38 dna:chromosome chromosome:CanFam3.1:38:1:23914537:1 REF
>X dna:chromosome chromosome:CanFam3.1:X:1:123869142:1 REF
>MT dna:chromosome chromosome:CanFam3.1:MT:1:16727:1 REF
>JH373233.1 dna:scaffold scaffold:CanFam3.1:JH373233.1:1:2660953:1 REF
>JH373234.1 dna:scaffold scaffold:CanFam3.1:JH373234.1:1:1881673:1 REF
>JH373235.1 dna:scaffold scaffold:CanFam3.1:JH373235.1:1:1415205:1 REF
>JH373236.1 dna:scaffold scaffold:CanFam3.1:JH373236.1:1:1067467:1 REF
>JH373238.1 dna:scaffold scaffold:CanFam3.1:JH373238.1:1:881102:1 REF
```
and it goes on...

Here is the first 45 lines of the output for `samtools view -H ../BAM/BH004_ReSorted.bam | grep '@SQ'`
```
@SQ     SN:1    LN:122678785
@SQ     SN:10   LN:69331447
@SQ     SN:11   LN:74389097
@SQ     SN:12   LN:72498081
@SQ     SN:13   LN:63241923
@SQ     SN:14   LN:60966679
@SQ     SN:15   LN:64190966
@SQ     SN:16   LN:59632846
@SQ     SN:17   LN:64289059
@SQ     SN:18   LN:55844845
@SQ     SN:19   LN:53741614
@SQ     SN:2    LN:85426708
@SQ     SN:20   LN:58134056
@SQ     SN:21   LN:50858623
@SQ     SN:22   LN:61439934
@SQ     SN:23   LN:52294480
@SQ     SN:24   LN:47698779
@SQ     SN:25   LN:51628933
@SQ     SN:26   LN:38964690
@SQ     SN:27   LN:45876710
@SQ     SN:28   LN:41182112
@SQ     SN:29   LN:41845238
@SQ     SN:3    LN:91889043
@SQ     SN:30   LN:40214260
@SQ     SN:31   LN:39895921
@SQ     SN:32   LN:38810281
@SQ     SN:33   LN:31377067
@SQ     SN:34   LN:42124431
@SQ     SN:35   LN:26524999
@SQ     SN:36   LN:30810995
@SQ     SN:37   LN:30902991
@SQ     SN:38   LN:23914537
@SQ     SN:4    LN:88276631
@SQ     SN:5    LN:88915250
@SQ     SN:6    LN:77573801
@SQ     SN:7    LN:80974532
@SQ     SN:8    LN:74330416
@SQ     SN:9    LN:61074082
@SQ     SN:MT   LN:16727
@SQ     SN:X    LN:123869142
@SQ     SN:JH373233.1   LN:2660953
@SQ     SN:JH373234.1   LN:1881673
@SQ     SN:JH373235.1   LN:1415205
@SQ     SN:JH373236.1   LN:1067467
@SQ     SN:JH373238.1   LN:881102
```
and it goes on...

**Comment by GiterKR** [comment]
Hi,

I issue was that chromosome order doesn't match between the reference sequence and the bam file. I fixed that by reordering my bam file and GRIDSS ran further this time. However, it is now giving me a different error.

Here is the command I used:
```
../../gridss.sh -r ../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa \
				-o VCF/BH004_grids.vcf.gz \
				-a BAM/BH004_grids.bam \
				--labels BH004 \
				../BAM/BH004_ReReSorted_ReOrdered.bam
```

Here is the output displayed:
```
Sun Mar 21 22:23:35 EDT 2021: Full log file is: ./gridss.full.20210321_222335.node204.hpc.local.240282.log
which: no time in (/usr/bin)
Sun Mar 21 22:23:35 EDT 2021: Not found /usr/bin/time
Sun Mar 21 22:23:35 EDT 2021: Using GRIDSS jar /home/rajk/gridss-2.11.0-gridss-jar-with-dependencies.jar
Sun Mar 21 22:23:35 EDT 2021: Using reference genome "../Reference/Canis_lupus_familiaris.CanFam3.1.dna.toplevel.fa"
Sun Mar 21 22:23:36 EDT 2021: Using output VCF VCF/BH004_grids.vcf.gz
Sun Mar 21 22:23:36 EDT 2021: Using assembly bam  BAM/BH004_grids.bam
Sun Mar 21 22:23:36 EDT 2021: Using 8 worker threads.
Sun Mar 21 22:23:36 EDT 2021: Using no blacklist bed. The encode DAC blacklist is recommended for hg19.
Sun Mar 21 22:23:36 EDT 2021: Using JVM maximum heap size of 30g for assembly and variant calling.
Sun Mar 21 22:23:36 EDT 2021: Using input file ../BAM/BH004_ReReSorted_ReOrdered.bam
Sun Mar 21 22:23:36 EDT 2021: label is BH004
Sun Mar 21 22:23:36 EDT 2021: Found /opt/software/R/4.0.2/bin/Rscript
Sun Mar 21 22:23:36 EDT 2021: Found /opt/software/samtools/1.11/bin/samtools
Sun Mar 21 22:23:36 EDT 2021: Found /etc/alternatives/java_sdk_1.8.0/bin/java
Sun Mar 21 22:23:36 EDT 2021: Found /opt/software/bwa/0.7.10/bin/bwa
Sun Mar 21 22:23:36 EDT 2021: samtools version: 1.11+htslib-1.11
Sun Mar 21 22:23:36 EDT 2021: R version: R scripting front-end version 4.0.2 (2020-06-22)
Sun Mar 21 22:23:36 EDT 2021: bwa Version: 0.7.10-r789
which: no time in (/usr/bin)
Sun Mar 21 22:23:36 EDT 2021: bash version: GNU bash, version 4.2.46(2)-release (x86_64-redhat-linux-gnu)
Sun Mar 21 22:23:36 EDT 2021: java version: openjdk version "1.8.0_242" OpenJDK Runtime Environment (build 1.8.0_242-b08)       OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode) 
Sun Mar 21 22:23:36 EDT 2021: Max file handles: 4096
Sun Mar 21 22:23:36 EDT 2021: Running GRIDSS steps: setupreference, preprocess, assemble, call,
Sun Mar 21 22:23:36 EDT 2021: Start pre-processing      ../BAM/BH004_ReReSorted_ReOrdered.bam
Sun Mar 21 22:23:36 EDT 2021: Running   CollectGridssMetrics    ../BAM/BH004_ReReSorted_ReOrdered.bam   first 10000000 records
Sun Mar 21 22:23:53 EDT 2021: Running   CollectGridssMetricsAndExtractSVReads|samtools  ../BAM/BH004_ReReSorted_ReOrdered.bam
Sun Mar 21 22:31:08 EDT 2021: Running   PreprocessForBreakendAssembly|samtools  ../BAM/BH004_ReReSorted_ReOrdered.bam
Sun Mar 21 22:56:47 EDT 2021: Complete pre-processing   ../BAM/BH004_ReReSorted_ReOrdered.bam
Sun Mar 21 22:56:47 EDT 2021: Start assembly     BAM/BH004_grids.bam
Sun Mar 21 22:56:47 EDT 2021: Running   AssembleBreakends        BAM/BH004_grids.bam    job 0   total jobs 1
"$timecmd java -Xmx$jvmheap $jvm_args -Dgridss.output_to_temp_file=true -cp $gridss_jar gridss.AssembleBreakends JOB_INDEX=$jobindex JOB_NODES=$jobnodes TMP_DIR=$workingdir WORKING_DIR=$workingdir REFERENCE_SEQUENCE=$reference WORKER_THREADS=$threads $input_args $blacklist_arg $config_args $picardoptions $readpairing_args O=$assembly" command completed with exit code 1.
*****
The underlying error message can be found in ./gridss.full.20210321_222335.node204.hpc.local.240282.log.
*****
```

The text in the log file is in the attached file:
[gridss.full.20210321_222335.node204.hpc.local.240282.log](https://github.com/PapenfussLab/gridss/files/6179529/gridss.full.20210321_222335.node204.hpc.local.240282.log)

Appreciate any help in fixing this.

**Comment by d-cameron** [comment]
I just released v2.11.1 which includes this fix.

**Comment by GiterKR** [comment]
Thank you very much, I am able to complete running GRIDSS with this new version.
